appserver/web/weld-integration/src/main/java/org/glassfish/weld/WeldDeployer.java

```diff
@@ -110,13 +110,7 @@
 import org.glassfish.web.deployment.descriptor.ServletFilterDescriptor;
 import org.glassfish.web.deployment.descriptor.ServletFilterMappingDescriptor;
 import org.glassfish.weld.connector.WeldUtils;
-import org.glassfish.weld.services.EjbServicesImpl;
-import org.glassfish.weld.services.ExternalConfigurationImpl;
-import org.glassfish.weld.services.InjectionServicesImpl;
-import org.glassfish.weld.services.NonModuleInjectionServices;
-import org.glassfish.weld.services.ProxyServicesImpl;
-import org.glassfish.weld.services.SecurityServicesImpl;
-import org.glassfish.weld.services.TransactionServicesImpl;
+import org.glassfish.weld.services.*;
```

Comment:
Whe should agree on some rule how many imports should be replaced with asterisk ... I have 20 as default ... and I would not like it under 10. Only minor comment here, does not block PR.

Reply:
Yeah IntelliJ defaults to something like 5 which I think is way too low.
We don't currently have one set, but I agree we should agree one.

Personally I hate wildcard imports in all situations.
You pretty much **never** need to import everything, and IDEs can auto-hide imports anyway so the "readability" argument is moot.

Reply:
Yep, SonarQube also has "too many imports" rule ... so usually asterisk should be unreachable :)

Reply:
It was set to 9 :). I'll turn it up a bit.